lib/site-inspector/checks/accessibility.rb

```diff
@@ -0,0 +1,32 @@
+require 'json'
+
+class SiteInspector
+  class Endpoint
+    class Accessibility < Check
+      
+      def section508
+        pa11y('Section508')
+      end
+      
+      def wcag2a
+        pa11y('WCAG2A')
+      end
+      
+      def wcag2aa
+        pa11y('WCAG2AA')
+      end
+      
+      def wcag2aaa
+        pa11y('WCAG2AAA')
+      end
+      
+      private
+      
+      def pa11y(reporter)
+        json_string = `pa11y https://18f.gsa.gov -s #{reporter} -r json`
```

Comment:
I'd use popen here, rather than backticks. That way we can capture error messages, and raise errors on non-zero statuses. See https://github.com/benbalter/word-to-markdown/blob/master/lib/word-to-markdown.rb#L66-L70 for an example.